File: sololearn/popsicles/popsicles.py
# Made by jisumov

siblings = int(input()) # Ask the amount of siblings
popsicles = int(input()) # Ask the amount of popsicles

if popsicles % siblings == 0: # Check if the amount of popsicles can be given away equally to the siblings
    print("give away")
else:
    print("eat them yourself")

File: sololearn/popsicles/popsicles.cpp
/* Made by jisumov */
#include <iostream>

using namespace std;

int main() {

    int siblings, popsicles;
    cin >> siblings >> popsicles;
    /* Ask the amount of siblings and popsicles */
    if(popsicles%siblings==0) {
        /* Check if the amount of popsicles can be given away equally to the siblings */
        cout << "give away";
    } else {
        cout << "eat them yourself";
    }

    return 0;
    
}